manifests/artifactory.pp

```diff
@@ -71,44 +65,29 @@
     fail("archive::artifactory[${name}]: \$name or \$archive_path must be an absolute path!") # lint:ignore:trailing_comma
```

Comment:
Odd lint ignore here. Also wondering if we should move the above line to just ```Stdlib::Absolutepath```

Reply:
I tried removing the lint, and puppet-lint *does* complain.  A false positive I guess.
Maybe I should use `assert_type` instead.

Reply:
I've updated this code to use assert_type and Stdlib::Absolutepath